crates/anvil/src/eth/backend/executor.rs

```diff
@@ -318,10 +320,10 @@ impl<DB: Db + ?Sized, V: TransactionValidator> Iterator for &mut TransactionExec
         }
 
         let exec_result = {
-            let mut evm = new_evm_with_inspector(&mut *self.db, env, &mut inspector, self.odyssey);
-            if let Some(factory) = &self.precompile_factory {
-                inject_precompiles(&mut evm, factory.precompiles());
-            }
+            let mut evm = new_evm_with_inspector(&mut *self.db, &env, &mut inspector);
+            // if let Some(factory) = &self.precompile_factory {
+            //     inject_precompiles(&mut evm, factory.precompiles());
+            // }
```

Comment:
Possibly related: https://github.com/alloy-rs/evm/pull/71

cc @yash-atreya

Reply:
should be merged soon